python3.6/Dockerfile

```diff
@@ -40,30 +43,30 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 # Create working directory
-ENV WORKING_DIR=/opt/invenio
-ENV INVENIO_INSTANCE_PATH=${WORKING_DIR}/var/instance
```

Comment:
If an instance is created with cookiecutter-invenio-instance using a name different than `invenio` this Dockerfile won't work, added the mandatory `ARG PROJECT_NAME` to address this.